addon/lib/document.js

```diff
@@ -127,6 +127,9 @@ export default Base.extend({
       console.debug(`${label}:`, any);
       return any;
     });
+    documentJexl.addTransform("min", (arr) => Math.min(...arr));
+    documentJexl.addTransform("max", (arr) => Math.max(...arr));
+    documentJexl.addTransform("round", (num) => Math.round(num));
```

Comment:
You should handle what happens if there is a non number as input. I don't think you want `NaN` in your JEXL. Those should return `null` when failing as do the other transforms.

Reply:
Updated as requested